packages/notify-client/src/types/client.ts

```diff
@@ -7,6 +7,12 @@ import { Logger } from "pino";
 
 import { INotifyEngine } from "./engine";
 
+interface ImageUrl {
```

Comment:
Nit: since the structure represents multiple URLs the naming should reflect it.

Also type assignments below are already `imageUrls: ImageUrl` so makes sense to align.
```suggestion
interface ImageUrls {
```